### PR TITLE
Update ext.html.erb

### PR DESCRIPTION
**Pull request by packtpartner**
Added a new book listing and updated a book cover image link for another book.

### CODE REVIEW

File: app/views/doc/ext.html.erb [hunk 1]
```diff
@@ -90,6 +90,13 @@
   <div class='two-column'>
     <div class='column-left'>
       <ul class='books-list'>
+        <li>
+         <a href="https://www.packtpub.com/application-development/git-essentials-second-edition"><img src="https://www.packtpub.com/sites/default/files/B06881.png" width="113" height="140"></a>
+         <h4><a href="https://www.packtpub.com/application-development/git-essentials-second-edition</a>Git Essentials - Second Edition</h4>
```
Comment (pedrorijo91) [inline app/views/doc/ext.html.erb]: <a> element is not properly closed.

should be something like:

```
<h4><a href="https://www.packtpub.com/application-development/git-essentials-second-edition">Git Essentials - Second Edition</a></h4>
```

File: app/views/doc/ext.html.erb [hunk 2]
```diff
@@ -90,6 +90,13 @@
   <div class='two-column'>
     <div class='column-left'>
       <ul class='books-list'>
+        <li>
```
Comment (pedrorijo91) [inline app/views/doc/ext.html.erb]: please, don't add your book to the top of the list. While there's no explicit ordering, it seems unlikely to have some other book than "Pro Git" on the top of the list

File: app/views/doc/ext.html.erb (hunk 1 above)
Comment (pedrorijo91) [inline app/views/doc/ext.html.erb]: also, getting a lot of 503 on packtpub.com...

**Comment by packtpartner** [comment]
Thank you for the feedback. I'll work on the changes right away.

**Comment by pedrorijo91** [comment]
migrated to https://github.com/git/git-scm.com/pull/1087

**Pull request closed without merging**